src/views/factor-verify/EmailMagicLinkForm.js

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+/* eslint complexity: [2, 7] */
+define(['okta', 'q', 'views/mfa-verify/PassCodeForm'], function (Okta, Q, PassCodeForm) {
+
+  var subtitleTpl = Okta.Handlebars.compile('({{subtitle}})');
+  var _ = Okta._;
+  var API_RATE_LIMIT = 30000; //milliseconds
+
+  return PassCodeForm.extend({
+    className: 'factor-verify-passcode',
+    initialize: function () {
+      // for FACTOR_REQUIRED with email magic link we dont need to show otp code input field and verify button
+      var form = this;
+      this.title = this.model.get('factorLabel');
+      //TODO: Temp fix for demo. FACTOR_REQUIRED after sign up is missing the profile object in API response
```

Comment:
- any jira for the todo?
- feels like we create another derived field in `appState` to read `lastAuthResponse.login`?

Reply:
Actually the TODO is on the server side once the API returns the right response, we should just be able to do `var email = this.model.get('email')` to get the email.

Reply:
at that point, do we need to remove the else condition (` || ...`)

Reply:
Yes https://oktainc.atlassian.net/browse/OKTA-211618

Reply:
please put in the comment as well